packages/components/src/styles/media/media.stories.ts

```diff
@@ -1,21 +1,25 @@
 import '../../solid-components';
-
+import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 
 const { argTypes, parameters } = storybookDefaults('sd-media');
 const { overrideArgs } = storybookHelpers('sd-media');
 const { generateTemplate } = storybookTemplate('sd-media');
 
 /**
- * Add a media caption to the bottom of an image.
+ * Used to display an image or a video preview.
+ *
+ *  * **Related templates**:
+ * - [Media](?path=/docs/templates-media--docs)
  */
 export default {
   title: 'Styles/sd-media',
+  tags: ['!dev'],
   component: 'sd-media',
   parameters: {
     ...parameters
   },
-  args: overrideArgs({ type: 'slot', name: 'default', value: 'Lorem Ipsum' }),
+  args: overrideArgs({ type: 'slot', name: 'default', value: 'Default' }),
```

Comment:
We are missing a description for default section. Can you check other components descriptions you use in code and adapt?

Reply:
Default section will be hidden therefore I believe there is no need for a description